mbta.rb

```diff
@@ -0,0 +1,32 @@
+
+
+
+puts "Hello, please tell me where you're starting from..."
```

Comment:
Idiomatic ruby would leave off the parends and spaces: `origin_stop = gets.chomp`